linkerd/core/src/main/scala/io/buoyant/linkerd/Linker.scala

```diff
@@ -104,24 +105,33 @@ object Linker {
     admin: Option[AdminConfig]
   ) {
 
-    def mk(defaultTelemeter: Telemeter = NullTelemeter): Linker = {
+    def mk(): Linker = {
       // At least one router must be specified
       if (routers.isEmpty) throw NoRoutersSpecified
 
-      val telemeters = telemetry match {
-        case None => Seq(defaultTelemeter)
-        case Some(telemeters) => telemeters.map {
-          case t if t.disabled =>
-            val msg = s"The ${t.getClass.getCanonicalName} telemeter is experimental and must be " +
-              "explicitly enabled by setting the `experimental' parameter to `true'."
-            throw new IllegalArgumentException(msg) with NoStackTrace
-          case t => t.mk(Stack.Params.empty + param.LinkerConfig(this))
-        }
+      val metrics = MetricsTree()
+
+      val telemeterConfigs = telemetry match {
+        case Some(telemeters) if telemeters.exists(_.kind == "io.l5d.commonMetrics") =>
```

Comment:
I think I'm going to revert this change to the way the default telemeter is handled.  My motivation was that it's really easy to mistakenly not configure a telemeter that provides `/admin/metrics.json` in which case the admin dashboard does not work.  In some sense, the admin dashboard should REQUIRE a telemeter that can provide that endpoint.  But I think what I have here is too weird.  

I'll revert this part and think on it some more.